src/components/Form.js

```diff
@@ -1,19 +1,42 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/Books';
 import form from './Form.module.css';
 
 function Form() {
+  let inputName;
+  let inputCategory;
+
+  const dispatch = useDispatch();
+
   return (
     <div className="ml-x">
       <h2 className={form.desc}>ADD NEW BOOK</h2>
```

Comment:
Please ensure that you have a local state with  `title`, `author` according to the project requirement in your `New Book React Component`.
   - should have your inputs that updates the local state every time you update (`onChange`)

Reply:
hi @micheaol. I'm still not clear about this requirement could you explain a bit further thanks 😃

Reply:
same thing you did with your own project. except the requirements were not the same. 
![react-bookstore_index js at main · micheaol_react-bookstore - Google Chrome 3_29_2022 7_59_50 AM](https://user-images.githubusercontent.com/40201915/160552759-3a2e591c-0304-4114-af8b-aa8d80cfc17a.png)

Reply:
@harlexkhal ,

Sure, please find below how to declare a local state using hooks because you have a functional component:
![image](https://user-images.githubusercontent.com/10332499/160554036-f5cdc3b6-6a63-4278-a7d0-950524910e06.png).

Please find below your project requirement:
![image](https://user-images.githubusercontent.com/10332499/160554393-d4701abe-a475-4f1e-9791-422b6e18348e.png)

I hope this helps.

Reply:
oh, I'm supposed to use the regular React hooks. Alright Thanks, for clearing that out. 👍